### DISCUSSION TITLE
2FA for login

### DISCUSSION THREAD

**Discussion by mucan54** [andreapollastri/cipi#352] [Ideas]
I think it is necessary to have two factor auth for admin login.

**Comment by andreapollastri** [comment]
Hi! In version 4 Laravel 9 and Laravel Jetstream with 2FA will be included!

**Comment by andreapollastri** [comment]
Hello guys, it the last 3 years the world has been changed with AI and with new deployments needs so a new "Laravel Control Panel" is not the real solution for today's world. I have created a new stable version of Cipi full of features and based on a powerful and light CLI. If you would like to try it, here you can find all informations: https://cipi.sh/
